tests/foreman/ui/test_errata.py

```diff
@@ -840,30 +830,47 @@ def test_positive_list_permission(
     :expectedresults: Check that the new user is able to see errata for one
         product only.
     """
-    module_org = module_org_with_parameter
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=function_sca_manifest_org.id,
+        product=PRDS['rhel8'],
+        repo=REPOS['rhst8']['name'],
+        reposet=REPOSET['rhst8'],
+        releasever='None',
+    )
+    rh_repo = module_target_sat.api.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
+    custom_repo = module_target_sat.api.Repository(
+        url=CUSTOM_REPO_URL, product=function_product
+    ).create()
+    custom_repo.sync()
+    # create role with access only to 'RHEL8' RedHat product
     role = module_target_sat.api.Role().create()
     module_target_sat.api.Filter(
-        organization=[module_org],
+        organization=[function_sca_manifest_org],
         permission=module_target_sat.api.Permission().search(
             query={'search': 'resource_type="Katello::Product"'}
         ),
         role=role,
-        search='name = "{}"'.format(PRDS['rhel']),
+        search='name = "{}"'.format(PRDS['rhel8']),
```

Comment:
Can we update this to use fstring instead?
```suggestion
        search=f'name = "{PRDS['rhel8']}"',
```

Reply:
@Gauravtalreja1 this format for fstring fails precommit since there are 3 distinct string literals.
Propose this format if we want this suggestion
`search=f'name = \'{PRDS["rhel8"]}\''`

Reply:
Apologies actually none of these suggestions work while also passing pre-commit.
Propose we use the original format